x-pack/plugins/siem/public/components/timeline/body/column_headers/events_select/index.tsx

```diff
@@ -60,7 +56,6 @@ export const EventsSelect = pure<Props>(({ checkState, timelineId }) => {
         itemClassName="eventsSelectItem"
         onChange={noop}
         options={getEventsSelectOptions()}
-        valueOfSelected={''}
```

Comment:
@elastic/secops The updates to `EuiSuperSelect` requires that the `value` properties of the `options` prop match the `valueOfSelected`. In this usage the `options['value']` prop is of type `EventsSelectAction` or `'select-all' | 'select-none' | 'select-pinned' | 'select-unpinned' | 'pin-selected' | 'unpin-selected'`, which `''` obviously doesn't meet. Can you please verify that removing this prop doesn't break anything?

Reply:
Thanks @spalger , I found some other warning from EUI and some test case failed when I was running this branch, should we fix that in this branch as well?